apps/android/app/src/main/java/chat/simplex/app/model/ChatModel.kt

```diff
@@ -1202,6 +1207,25 @@ data class ChatItem (
         file = null
       )
     }
+
+    private const val TEMP_DELETED_CHAT_ITEM_ID = -1L
+    val defaultDeleted: ChatItem
+      get() = ChatItem(
+        chatDir = CIDirection.DirectRcv(),
+        meta = CIMeta(
+          itemId = TEMP_DELETED_CHAT_ITEM_ID,
+          itemTs = Clock.System.now(),
+          itemText = generalGetString(R.string.deleted_description),
+          itemStatus = CIStatus.RcvRead(),
+          createdAt = Clock.System.now(),
+          itemDeleted = true,
+          itemEdited = false,
+          editable = false
+        ),
+        content = CIContent.RcvDeleted(deleteMode = CIDeleteMode.cidmBroadcast),
```

Comment:
it probably would be cleaner if it was Rcv or SndDeleted depending on direction.. though it's a hack anyway and this would probably require passing additional parameters so overall might not be worth it..